### ISSUE TITLE
Running in background

### ISSUE THREAD

**Issue by JoeRourke123** [thebaselab/codeapp#52]
As iOS prevents tasks running in the background, a web server cannot be accessed while Code App is in the background (so must either have split screen or switch back and forth between requests). Some apps, such as Terminus, get around this by doing a frequent location checks while the app is in the background, which allows the app to continue running tasks - could a similar solution be used here?

**Comment by bummoblizard** [comment]
I will look into this. But it’s rather hacky and could be frustrating for users with privacy concerns. Another solution is to have a built web browser in the app, if that’s what you need.

**Comment by JoeRourke123** [comment]
> I will look into this. But it’s rather hacky and could be frustrating for users with privacy concerns. Another solution is to have a built-in web browser within the app, if that’s what you need.

Yes the ability to open browser windows within the app would be really cool - thank you!

**Comment by sylvoie** [comment]
Hello,

I coding app with django-python and I have the same trouble. Please add a me too! 

I used PlayJS and they have a built-in web browser within the app, and it’s working fine.

Thank’s

**Comment by sylvoie** [comment]
Hello,

Please, let me add some needs.

I like the « preview in safari » feature. That feature is really useful and needed to test web development, like Django development. For me, this feature is really important because it lets us test and code inside the app, at the same time reduce the need of « background execution ». 

An other important behaviour, the context in web browser need to stay each time we minimize/restore this web browser to let us test without starting from the beginning. 

We need URL bar too to enter the web path of our web app. I tried to connect my Django web server without success. I putted the path of my web server in the html file, but always without success. Even if I coded « <a href=«http://127.0.0.1:8000»> in my html file, connecting wasn’t possible. I don’t know how to do.

Thank’s for your help
